project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.3.3
```

Comment:
Well, I don't really want to downgrade to 1.2.8.
There must be a way to make things work with 1.3.3 as well...